tabs/src/translations/en-US.json

```diff
@@ -1,6 +1,58 @@
 {
     "translation": {
 
+        "Query Runner": {
+            "Key": "Key",
+            "Value": "Value",
+            "Run query": "RUN QUERY",
+            "Request body": "Request Body",
+            "Request headers": "Request Headers",
+            "Response body": "Response Body",
+            "Response headers": "Response Headers",
+            "Response": "Response",
+            "Warning": "To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
```

Comment:
Revise the text of the warning: "You must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article}." We avoid using "here" as link text due to accessibility.